package_creator.py

```diff
@@ -23,6 +25,53 @@
 IMAGE_PREFIX = 'data:image/png;base64,'
 
 
+def write_yaml_with_docker(output_path, yml_text, yml_data, script_obj):
+    """Write out the yaml file taking into account the dockerimage45 tag.
+    If it is present will create 2 integration files
+    One for 4.5 and below and one for 5.0.
+
+    Arguments:
+        output_path {str} -- output path
+        yml_text {str} -- yml text
+        yml_data {dict} -- yml object
+        script_obj {dict} -- script object
+
+    Returns:
+        dict -- dictionary mapping output path to text data
+    """
+    output_map = {output_path: yml_text}
+    if 'dockerimage45' in script_obj:
+        # we need to split into two files 45 and 50. Current one will be from version 5.0
+        yml_text = re.sub(r'^\s*dockerimage45:.*\n?', '', yml_text, flags=re.MULTILINE)  # remove the dockerimage45 line
+        yml_text45 = yml_text
+        if 'fromversion' in yml_data:
+            yml_text = re.sub(r'^fromversion:.*$', 'fromversion: 5.0.0', yml_text, flags=re.MULTILINE)
```

Comment:
why you replace fromversion with 5.0.0? the same toversion with4.5.9?
what if toversion was 4.0.0?
the same for fromversion in the future could be 5.5.0

Reply:
You are correct. I assumed the dockerimage45 will be used only when needed. Will add a validation.